package.json

```diff
@@ -21,10 +22,10 @@
     "eslint": "8.28.0",
     "eslint-config-next": "13.0.4",
     "framer-motion": "^10.12.4",
-    "next": "13.0.4",
+    "next": "^13.4.3",
```

Comment:
why did we update next version here ?

Reply:
When i was installing next js i mistakenly updated to the latest version will change it back to 13.0.4